### DISCUSSION TITLE
Menu Entry Swap for Annette

### DISCUSSION THREAD

**Discussion by ChopinDavid** [runelite/runelite#17396] [Ideas]
Annette is the lady who stores your noted nets at drift net fishing.

There should be a menu entry swap to change her "Talk-to" to "Nets".

I could code this if need be. Just need it to probably become a proper issue before I make a PR.

**Comment by Bram91** [comment]
The Menu Entry Swapper, through various config options, allows you to hold shift+right click on items/NPCs/objects and configure new default left and shift click options on them 
![right_click_with_cancel](https://github.com/runelite/runelite/assets/7499230/0ed36b57-8596-4a12-8551-ee94d8608958)

**Reply by ChopinDavid** [reply]
Sweeeet, thank you!
